docs/context.rst

```diff
@@ -2,11 +2,13 @@ Context & Breadcrumbs
 =====================
 
 The Java SDK implements the idea of a "context" to support attaching additional
-information to a specific user request, such as breadcrumbs.
+information to events, such as breadcrumbs. A context may refer to a single
+request to a web framework, to the entire lifetime of an Android application,
+or something else that better suits your application's needs.
 
-Unlike some other platforms, Java has no single definition of what "context" means.
-For this reason a specific implementation must be chosen depending on what framework
-you are using. By default Raven uses a ``ThreadLocalContextManager`` that maintains
+There is no single definition of context that applies to every application,
+for this reason a specific implementation must be chosen depending on what your
+application does. By default Raven uses a ``ThreadLocalContextManager`` that maintains
```

Comment:
It's more meta than concurrency model, since Android uses `SingletonContextManager`... :(